### DISCUSSION TITLE
Vlang.io VS challenges.cloudflare.com VS DDoS

### DISCUSSION THREAD

**Discussion by Wajinn** [vlang/v#20901] [General]
Hey guys,

You might want to check your settings or talk to people at cloudflare, because your website has become harder to reach. Also the cloudflare challenge is famously worthless. It will just push lots of visitors attempting to reach the site into an endless circle where it never gives access. This kind of site availability problem has probably already done significant damage to your public outreach, as it has quietly and unsuspectingly done to many other sites under cloudflare.

**Comment by medvednikov** [comment]
This is a temporary measure until the DDoS is over.

**Comment by medvednikov** [comment]
Seems to be ok now, I've disabled the challenge. It lasted about 1.5 days.

**Reply by Wajinn** [reply]
Sad to read that the site may have been experiencing this type of malicious or criminal attack. Just wanted to note what some of V's visitors and users might be experiencing (cloudflare's endless challenge) as a result of/on top of the DDoS. Perhaps there are some other tweaks that can be worked out with the settings or cloudflare, to help with site availability in those situations.

Glad to read that the situation is under control.
